connect/eaas/core/inject/common.py

```diff
@@ -54,3 +55,11 @@ def get_logger(
 
 def get_config(x_connect_config: str = Header('{}')):
     return json.loads(x_connect_config)
+
+
+def get_correlation_id(connect_correlation_id: str = Header()):
```

Comment:
It does not make much sense to me to have this function here, it is an utility function. I prefer to have it in utils.py
Moreover no need to declare it has fastapi `Header`.